alvr/client_core/src/decoder.rs

```diff
@@ -61,26 +61,23 @@ pub fn create_decoder(config_nal: Vec<u8>) {
                     .send(alvr_sockets::ClientControlPacket::RequestIdr)
                     .ok();
             }
-
-            unsafe { crate::notifyNewDecoder() };
         }
     }
 }
 
-pub extern "C" fn push_nal(buffer: *const c_char, length: i32, timestamp_ns: u64) {
+pub fn push_nal(buffer: *const u8, length: i32, timestamp_ns: u64) {
```

Comment:
If this is not called from the C++ side anymore, prefer using `Vec<u8>` for the buffer instead of pointer + size.